src/hotspot/share/memory/guardedMemory.hpp

```diff
@@ -42,9 +42,10 @@
  * |Offset             | Content              | Description    |
  * |------------------------------------------------------------
  * |base_addr          | 0xABABABABABABABAB   | Head guard     |
- * |+16                | <size_t:user_size>   | User data size |
- * |+sizeof(uintptr_t) | <tag>                | Tag word       |
- * |+sizeof(uintptr_t) | <tag2>               | Tag word       |
+ * |+16                | padding              | For alignment  |
```

Comment:
Is `+sizeof(Guard)` or `+GUARD_SIZE` an alternative to `+16`. 16 is a little out of the blue for me.

Reply:
Fixed